pallets/parachain-staking/src/lib.rs

```diff
@@ -114,7 +115,7 @@ pub mod pallet {
 
 	/// Configuration trait of this pallet.
 	#[pallet::config]
-	pub trait Config: frame_system::Config {
+	pub trait Config: frame_system::Config + cumulus_pallet_parachain_system::Config {
```

Comment:
@Agusrodri  it creates an unnecessary coupling beetween pallets parachain-system and parachain-staking, it's a bad pattern. Please create a new Config associated type instead, something like `RelayChainBlockNumberProvider: Get<polkadot_parachain_primitives::primitives::RelayChainBlockNumber>`